css/css-backgrounds/border-image-shorthand-002.htm

```diff
@@ -1,16 +1,22 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-    <head>
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
         <title>CSS Test: Omitted values for 'border-image-outset' property in the 'border-image' shorthand</title>
-        <link rel="author" title="Microsoft" href="http://www.microsoft.com/" />
-        <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-shorthand" />
-        <meta name="flags" content="image" />
-        <meta name="assert" content="This test checks that the omitted 'border-image-outset' value in the shorthand property is set to the initial value." />
-        <style type="text/css">
+        <link rel="author" title="Microsoft" href="http://www.microsoft.com/">
+        <!--
+        Test rehabilitated by Gérard Talbot
+        2020-05-27
+        -->
+        <link rel="help" href="https://www.w3.org/TR/css-backgrounds-3/#the-border-image">
+        <link rel="match" href="reference/border-image-shorthand-002-ref.html">
+        <meta name="flags" content="">
+        <meta name="assert" content="This test checks that the omitted 'border-image-outset' value in the shorthand property is set to the initial value.">
+        <style>
             #test
             {
                 border: 20px double red;
```

Comment:
This needs a border-image-outset declaration here that will be overridden by the shorthand.

Reply:
Yes. Correct.